### PR TITLE
Css clean up

### LINKED ISSUE #343: Clean Up CSS
**Purpose**
Clean Up CSS 
The styling for this project is kind of all over the place. 

The goal of this update should be to simplify the CSS implementation in order to minimize duplicity and create a single source of truth. This will help with future development change, especially as we prepare to implement the design changes in the accessibility audit.

We will be leveraging the [Bulma Library](https://bulma.io/) for this project. 

**Proposed Solution**
Organize the Css into a single file and update the project's JSX in order to call the correct css classes. 

**Acceptance Criteria**
Please list the requirements the implementation of this feature should meet.
- [ ] CSS is located in a single readable file
- [ ] classNames are homogenized
- [ ] The new CSS file is implemented throughout the project and excessive code is removed
- [ ] Bulma is used to maximum effect (let's not reinvent any wheels).

### PR DESCRIPTION
**Pull request by SeHarlan**
### What issue is this solving?
#343 stage one
I've made all the buttons I could find have bulma classnames. Bulma has overrides we can use so if we want to change the hover behavior in all of our buttons we only need to implement this variable in the variables page, $button-text-hover-color.

most hard coded colors I found were changed to variables so they can all be changed from one place.

Next steps are to clean up the code I've commented out and figure out the best way to organize the CSS that we do want to keep, which I will try and keep to just structural stuff.

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? Bulma version has been upgraded to 9.0 so I believe so
Any special requirements to test?

### Please make sure you've attempted to meet the following coding standards
- [ ] Code builds successfully
- [ ] Code has been tested and does not produce errors
- [ ] Code is readable and formatted
- [ ] There isn't any unnecessary commented-out code
- [ ] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

**Review by beekman — DISMISSED** [review]
Looks good, nice job cleaning up and converting to bulma ui standards. Hopefully we can clean up the comments soon, as things won't actually be all that much cleaner until those comments are relegated to  git history.

**Comment by beekman** [comment]
I went ahead and took the leap to a single scss file, main.scss. Moved all scss from other files to replace their import statements. If this wasn't the intent, we can always revert the commits.

**Review by beekman — APPROVED** [review]

File: src/components/Button/button.scss [hunk 1]
```diff
@@ -1,73 +1,73 @@
 @import '../../scss/variables.scss';
 
-.styled-button {
-  position: relative;
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-  vertical-align: top;
+// .styled-button {
```
Comment (Chris-Boe) [inline src/components/Button/button.scss:3]: The reference to the deleted `variables.scss` above needs to be deleted to be able to run the app, as well as in all other files that reference it. Once it's gone, the variables will need to be imported then from `main.scss`

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
If these style changes are good and permanent, I think we should delete all the commented out CSS

**Comment by SeHarlan** [comment]
Sorry @beekman but I think I'm gonna revert a couple commit cause I don't think it's helpful to combine the variable, helper and main css files. Let's talk tonight about what we wanna do with all the structural CSS spread about in the component files, I think that's more what I had in mind when talking about streamlining the CSS files

**Comment by beekman** [comment]
@SeHarlan that's fine, I was just trying to follow your instructions to unify all the css definitions into a single file, it seemed strange to me as well. That clarification makes a lot more sense.

**Comment by SeHarlan** [comment]
@beekman Sorry about that, I'll try to be clearer in the future. There are a bunch of other CSS files other than those four that I a little more concerned about tbh.

**Comment by SeHarlan** [comment]
Commits reset and moved to css-clean-up branch and PR #382

**Pull request closed without merging**